### DISCUSSION TITLE
Developer Debug Mode

### DISCUSSION THREAD

**Discussion by sskit** [lllyasviel/Fooocus#3375] [Q&A] [answered]
Settings are not saved. How to make the settings be saved. In particular, "Save Metadata to Images" is disabled by default for some reason. How to enable and save this setting?

**Comment by mashb1t** [comment, marked as answer]
You can persist almost all if not all settings of the developer debug mode in the config.txt file. Please find the keys in the config_modification_tutorial.txt file.
